src/app/ecosystem-explorer/project-form/components/ProjectForm.tsx

```diff
@@ -0,0 +1,225 @@
+  categoryData: CategoryMap
+  subCategoryData: CategoryMap
+  initialValues: EcosystemProjectFormData
+}
+
+export function ProjectForm({
+  categoryData,
+  subCategoryData,
+  initialValues,
+}: ProjectFormProps) {
+  const yearOptions = getYearOptions('desc')
+  const categoryOptions = getOptionsFromObject(categoryData)
+  const subCategoryOptions = getOptionsFromObject(subCategoryData)
+
+  const form = useForm<EcosystemProjectFormData>({
+    resolver: zodResolver(EcosystemProjectFormSchema),
+    defaultValues: initialValues,
+  })
+  const isSubmitting = form.formState.isSubmitting
+
+  const submitForm = useSubmitEcosystemProjectForm()
+
+  return (
+    <ControlledForm<EcosystemProjectFormData>
+      form={form}
+      className="max-w-readable"
+      onSubmit={submitForm}
+    >
+      <FormSection
+        title="Personal Information"
+        description="We'll use this to verify your site updates and reach out to you in case of questions."
+      >
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="name"
+          label="Name"
+          placeholder="Your Name"
+          disabled={isSubmitting}
+        />
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="email"
+          label="Email"
+          placeholder="Your Email"
+          type="email"
+          disabled={isSubmitting}
+        />
+      </FormSection>
+
+      <FormSection title="Project Details">
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="projectName"
+          label="Project Name"
+          placeholder="Project Name"
+          disabled={isSubmitting}
+        />
+
+        <FormField
+          label="Which technology does your project utilize?"
+          error={form.formState.errors.tech?.root?.message}
+        >
+          <div className="flex flex-col gap-4">
+            <ControlledFormCheckbox<EcosystemProjectFormData>
+              name="tech.filecoin"
+              label="Filecoin"
+              disabled={isSubmitting}
+            />
+            <ControlledFormCheckbox<EcosystemProjectFormData>
+              name="tech.ipfs"
+              label="IPFS"
+              disabled={isSubmitting}
+            />
+          </div>
+        </FormField>
+
+        <ControlledFormListbox<EcosystemProjectFormData>
+          name="yearJoined"
+          label="What year did your project start using Filecoin or IPFS?"
+          placeholder="Select year"
+          options={yearOptions}
+          disabled={isSubmitting}
+          innerWidth="w-40"
+        />
+
+        <ControlledFormTextarea<EcosystemProjectFormData>
+          name="briefSummary"
+          label="Brief Summary"
+          placeholder="Write a brief summary of your project..."
+          characterLimit={BRIEF_CHARACTER_LIMIT}
+          disabled={isSubmitting}
+        />
+
+        <ControlledFormTextarea<EcosystemProjectFormData>
+          name="networkUseCase"
+          label="Network Use Case"
```

Comment:
```suggestion
          label="How do you use the Filecoin Network?"
```

 [Figma](https://www.figma.com/design/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?node-id=1231-24804&t=CKzEG4OJYmUxQYtb-4)